### ISSUE TITLE
RFID reading while eating

### ISSUE THREAD

**Issue by soostdijck** [psy0rz/meowton#2]
I saw your note about reading the RFID while eating. I have a couple "SureFeed Microchip Pet Feeder - MPF001". One of them broke a while ago and I took it apart. Indeed the method that they use is a giant wound coil inside the brace that the cat enters through.

I works quite well, and should do the trick for your RFID needs as well.

I can take some pics of the insides if you like.

**Comment by psy0rz** [comment]
Yeah, i have one petsafe catflap thats broken, that i can use to figure out how it works. But i have no time for extra projects currently :)
